rust/pact_mock_server_cli/src/server.rs

```diff
@@ -179,20 +183,27 @@ fn shutdown_resource(server_key: Arc<String>) -> WebmachineResource {
   }
 }
 
-fn mock_server_resource(output_path: Arc<Option<String>>) -> WebmachineResource {
+fn mock_server_resource(
+    output_path: Arc<Option<String>>,
+    server_manager: Arc<Mutex<ServerManager>>
+) -> WebmachineResource {
+    let server_manager1 = server_manager.clone();
```

Comment:
This is one of the things I hate about Rust! :-D

Reply:
Yes, it's not nice. The pattern does tend show up in closure-heavy code, which often makes me covert closures to functions, but I did not want to do a big rewrite here.